### PR TITLE
Fix/player 3832

### PR DESCRIPTION
**Pull request by zelibobla**
Added gulp webserver for debugging reasons.
Core fix: ignored 1009 'The VAST response document is empty' Google IMA SDK error returned when request for ads is cancelled.

### CODE REVIEW

File: js/google_ima.js [hunk 1]
```diff
@@ -1385,6 +1385,17 @@ require("../html5-common/js/utils/utils.js");
        */
       var _onImaAdError = privateMember(function(adError)
       {
+        if (adError &&
+          adError.type === 'adError' &&
+          adError.h &&
```
Comment (pilievOoyala) [inline js/google_ima.js]: Is this safe? It looks like you are accessing a the variable from the minified code which could change with their next release. There should be a function you can call to extract the ad error code.

Comment (pilievOoyala) [reply js/google_ima.js]: https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.AdError

just do aderror.getErrorCode()

Comment (zelibobla) [reply js/google_ima.js]: Thank you, Peter. Fixed this.

File: js/google_ima.js [hunk 2]
```diff
@@ -1385,6 +1385,16 @@ require("../html5-common/js/utils/utils.js");
...
       var _onImaAdError = privateMember(function(adError)
       {
+        if (adError &&
+          adError.type === 'adError' &&
+          adError.getErrorCode() === 1009 /*'The VAST response document is empty.'*/ ) {
+          // Empty response happens when 2 or more ads must be loaded simultaneously after playback seek ahead
+          // for 2 or more points of ads. In this case all prev ads loads are cancelled and the only last loaded.
+          // The requests for already cancelled ads seems to return empty VAST. Consider this is not an error
+          // @see https://jira.corp.ooyala.com/browse/PLAYER-3832 discussion
+          return;
```
Comment (aeng7) [inline js/google_ima.js]: Are we going to add some additional checks in the UI like @jpfranco mentioned in PLAYER-3832? If so, will this part still be needed?

**Pull request closed without merging**